docs/blog/2019-01-31-why-themes/index.md

```diff
@@ -41,7 +41,7 @@ Plugins have been a very successful lower-level abstraction for Gatsby. Themes i
 
 - People new to Gatsby that want to get started quickly without a difficult learning curve
 
-- People who have clear and common use-cases in mind for Gatsby (e.g. blog, ecommerce, documentation) and want to share the complexity of maintaining the underlying functionality with others — so pick a theme as the base for their site instead of starting from scratch
+- People who have clear and common use-cases in mind for Gatsby (e.g. blog, E-commerce, documentation) and want to share the complexity of maintaining the underlying functionality with others — so pick a theme as the base for their site instead of starting from scratch
```

Comment:
```suggestion
- People who have clear and common use-cases in mind for Gatsby (e.g. blog, e-commerce, documentation) and want to share the complexity of maintaining the underlying functionality with others — so pick a theme as the base for their site instead of starting from scratch
```